crowbar_framework/spec/models/service_object_spec.rb

```diff
@@ -84,7 +84,7 @@
       it "limits the number of elements in a role" do
         dns_proposal.elements["dns-client"] = ["admin"]
 
-        dns_service.stubs(:role_constraints).returns({ "dns-client" => { "count" => 0, "admin" => true } })
+        allow(dns_service).to receive(:role_constraints).and_return({ "dns-client" => { "count" => 0, "admin" => true } })
```

Comment:
Metrics/LineLength: Line is too long. [122/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.